pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[project]
+name = "arrayed-degradation-assay"
+version = "0.1.0"
+description = "Add your description here"
+authors = [
+    { name = "Albert Roethel", email = "albert@inceptive.team" }
+]
+dependencies = [
```

Comment:
For library dependencies, it could be good to be a bit more flexible on the version otherwise it will clash with other dependencies.

Reply:
Do you I can just drop all versions? All of these dependencies are equal to the ones that we currently use so on our end there should not be a problem with results being different.
Alternatively I can I for instance `>= version` for all dependencies.

Reply:
https://stackoverflow.com/questions/8795617/how-to-pip-install-a-package-with-min-and-max-version-range

I think we should fix the major version 1.x.x and ask for higher than what we use.